packages/Popover/src/hooks.ts

```diff
@@ -56,3 +56,18 @@ export function useMutationObserver<Value>(
 
   return value;
 }
+
+export function useElementNode<ElementType = HTMLElement>(): [
+  ElementType | null,
+  (node: ElementType | null) => void
+] {
+  const [element, setElement] = useState<ElementType | null>(null);
+
+  const elementRefCallback = useCallback(node => {
```

Comment:
useCallback is being memoized via the second arg. I'm a little confused. Also most of this is [inspired from the React docs](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node).

Reply:
🤦‍♂ 

I don't know why, but I was thinking of the empty array as saying, "this should fire every time", rather than only ever firing once. Ignore this, I was the one confused 😅